Dockerfile

```diff
@@ -59,4 +59,4 @@ COPY . .
 RUN adduser --disabled-password --gecos "OpenWPM"  openwpm
 
 # Alternatively, python3 could be used here
-CMD python demo.py
+CMD python crawler.py
```

Comment:
Do we want to change this to `crawler.py` in the main repo? Given that this script requires a bunch of environment variables to be set to run correctly, it seems like we should keep `demo.py` as the default?

Reply:
Sure, we can add the crawler.py command to the crawl yaml files

Reply:
Done